lib/tasks/data.rake

```diff
@@ -75,4 +76,137 @@ namespace :aleftng do
       end
     end
   end
+
+  def find_and_replace_in_text(find, replace, text)
+    #puts "Search and replace #{find.inspect} => #{replace.inspect}"
+    text.gsub!(find, replace)
+    return text
+  end
+
+  # TODO - Výnimka - iný typ otázky ako tieto základné
+  def find_question_type(text)
+    if text == "single-choice"
+      return "SingleChoiceQuestion"
+    elsif text == "multi-choice"
+      return "MultiChoiceQuestion"
+    elsif text == "answer-validator"
+      return "EvaluatorQuestion"
+    elsif text == "complement"
+      return "Complement"
+    end
+  end
+
+  def find_substring(text)
+    if text.include? "<correct>"
+      return true
+    else
+      return false
+    end
+  end
+
+  def import_Choice_questions(dir)
+    # Prečitanie súboru a vynechanie vypísania hlavičky pri každom zázname
+    parsed_file = CSV.read(dir, :headers => false)
+    parsed_file.each do |row|
+
+      # Predspracovanie
+      zero_version = row[4]
+      picture = row[1]
+      external_reference = row[0]
+      question_text = PandocRuby.new(row[5], :from => :docbook, :to => :markdown)
+      question_text = find_and_replace_in_text("\n", "", question_text.to_s)
```

Comment:
Tu mi PandocRuby pri konverzii vkladal nové riadky za každú ukončenú dvojicu <></> tak som to rušil. Ale ak sú niektoré otázky naformátované tak to bude problém.